File: Proyectos/Proyecto_2/backend/ruta-python/config/credentials.py
import os
from dotenv import load_dotenv

load_dotenv()

dbConfig = {
    'host': os.getenv('DB_HOST'),
    'user': os.getenv('DB_USER'),
    'password': os.getenv('DB_PASSWORD'),
    'database': os.getenv('DB_DATABASE'),
    'port': os.getenv('DB_PORT')
}

HOST_REDIS = os.getenv('HOST_REDIS')

File: Tareas/Tarea_7/Codigo_Fuente/ruta-go/server/config/credentials.go
package config

import (
	"fmt"
	"os"
	"paquetes/models"

	"github.com/joho/godotenv"
)

var DBConfig = models.DBEnv{}

func LoadEnv() {
	err := godotenv.Load()
	if err != nil {
		fmt.Println("Error loading .env file")
	}

	DBConfig.DB_HOST = os.Getenv("DB_HOST")
	DBConfig.DB_PORT = os.Getenv("DB_PORT")
	DBConfig.DB_DATABASE = os.Getenv("DB_DATABASE")
	DBConfig.DB_USER = os.Getenv("DB_USER")
	DBConfig.DB_PASSWORD = os.Getenv("DB_PASSWORD")
}